contrib/bradfitz/gomemcache/memcache/example_test.go

```diff
@@ -0,0 +1,20 @@
+package memcache_test
+
+import (
+	"context"
+
+	"github.com/bradfitz/gomemcache/memcache"
+	memcachetrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/bradfitz/gomemcache/memcache"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func Example() {
+	span, ctx := tracer.StartSpanFromContext(context.Background(), "example",
```

Comment:
Can we make up some better imaginary names than "_example_" for span and service? Let's think of a basic simplistic use-case or "hello world" example.